_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -371,18 +322,18 @@ This [HTML document][] has a [keyboard event][] [dispatched][] to an [event targ
 </html>
 ```
 
-#### Inapplicable Example 3
+#### Inapplicable Example 2
 
-This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [non-printable character][non-printable characters].
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] but it does not cause [changes in content][] unless the `getModifierState` returns `true`.
```

Comment:
This is indeed correct for testing points 2 and 3 of the Applicability. But I add to think a bit to convince myself of it… Not sure if we can update the description in a way that make it clearer without being too heavy 🤔 
(the think is that events that can be triggered either have `getModifierState("Control")` return `true` (failing 2nd point); or do not cause change in content (failing 3rd point). And it's not really possible to split these conditions in separate tests since any event can be fired by users…

So feel free to ignore this comment 😉

Reply:
The only way I can think to improve the description is to name conditions in the applicability list (e.g. "printable key", "no modifier", "changes content"), so they could be referenced here. Do you think it is worth it?

Reply:
No, this should be good.